README.md

```diff
@@ -1,20 +1,19 @@
-# Twilio React Video App
+# Twilio Video React App
 
-[![CircleCI](https://circleci.com/gh/twilio/twilio-video-app-react.svg?style=svg&circle-token=9d6b1e89d148181aaa6874c29849c730b8ca406d)](https://circleci.com/gh/twilio/twilio-video-app-react)
+[![CircleCI](https://circleci.com/gh/twilio/twilio-video-app-react.svg?style=shield&circle-token=9d6b1e89d148181aaa6874c29849c730b8ca406d)](https://circleci.com/gh/twilio/twilio-video-app-react)
```

Comment:
While we're at it, the token isn't needed anymore because its public. Can you change it to:
```
[![CircleCI](https://circleci.com/gh/twilio/twilio-video-app-react.svg?style=shield)](https://circleci.com/gh/twilio/twilio-video-app-react)
```

Reply:
Really? The GitHub project is public but CircleCI is private right?

Reply:
After testing I think you are correct! I can view status on circle but can't modify anything. Very nice.

Reply:
Done